documentation/grouping.html

```diff
@@ -1279,6 +1279,12 @@ <h2 id="pagination">Pagination</h2>
 so that a continuation given later will replace any shared state with a continuation given before.
 Also, when using the <code>continuations</code> annotation, always pass the <em>this</em> continuation as its first element.
 </p>
+<p>
+Note: Continuations work best when you have a stable ordering of hits - typically achieved by using
+<a href="ranking.html">ranking</a>, <a href="sorting.html"> or <a href="reference/grouping-syntax.html#order">ordering</a>
+ - possibly also with some additional tie-breaker like <a href="reference/rank-features.html#random">random.match</a>
+  to keep the order stable in case of duplicates.
```

Comment:
duplicates as in documents with the same rank score I assume? Please clarify this. Also I'm uncertain if you have sorting spec, the groups will still be ordered by relevance() by default.

Reply:
Confirmed that sorting doesn't help. Attempted to clarify what is meant by duplicates.